routes/movies-routes.js

```diff
@@ -0,0 +1,79 @@
+// starter code in both routes/celebrities.routes.js and routes/movies.routes.js
+const router = require("express").Router();
+
+const Movie = require('./../models/Movie.model')
+
+/* GET Movies page */
+router.get("/movies/create", (req, res) => res.render("./../views/movies/new-movie"))
+
+/*POST Movies page*/
+
+router.post('/movies/create', (req, res) => {
+
+    const { title, genre, plot, cast } = req.body
+
+    Movie
+        .create({ title, genre, plot, cast })
+        .then(() => res.redirect('/movies/list'))
+        .catch(err => console.log(err))
+})
+/*GET Movies list */
+router.get('/movies/list', (req, res) => {
+
+    Movie
+        .find()
+        .select('title')
+        .then(movies => res.render('./../views/movies/movies', { movies }))
+        .catch(err => console.log(err))
+})
```

Comment:
Te saltaste el paso del populate, por eso te aparecen ids.

```suggestion
router.get('/movies/list', (req, res) => {

    Movie
        .find()
        .select('title')
        .populate("cast")
        .then(movies => res.render('./../views/movies/movies', { movies }))
        .catch(err => console.log(err))
})
```